03_analysis/color_by_moiety.py

```diff
@@ -127,7 +136,8 @@ def main(in_dict, pickle_file, smi_files, out_prefix):
         prefix of the names of the output plots
 
     """
-    method_labels = list(in_dict.keys())
+#    method_labels = list(in_dict.keys())
+    method_labels =[k[:-2] if (k.endswith('.0') or k.endswith('.1')) else k for k in in_dict.keys()]
```

Comment:
Because I wanted to have only "OpenFF-X.Y" in the plots, I removed ".Z" from the labels. This customization was removed in b59b5b5 as it should be read in from the file